### PR TITLE
Add standard and run standard --fix

### PR DESCRIPTION
**Pull request by scwe**

### CODE REVIEW

File: src/color-web-sdk.js [hunk 1]
```diff
@@ -46,9 +46,9 @@ class ColorWebSDK {
   }
 
   async keepSession () {
-    if (!this.session) {return false}
+    if (!this.session) { return false }
     const pingInterval = this.session.pingInterval * MILIECONDS
-    return await setTimeout(this.updateSession.bind(this), pingInterval)
+    return setTimeout(this.updateSession.bind(this), pingInterval)
```
Comment (wojciechAdaszynski) [inline src/color-web-sdk.js]: maybe instead of mutual recursion we can use `setInterval` function

Comment (scwe) [inline src/color-web-sdk.js]: I can look into that in another PR... Or you can :heart:

Comment (psmarcin) [inline src/color-web-sdk.js]: I dont think we should use `setInterval` because sometime `pingInterval` can change...

**Comment by psmarcin** [comment]
@scwe conflicts

**Comment by scwe** [comment]
@psmarcin Fixed

**Pull request merged**